src/utils/fake-url.ts

```diff
@@ -5,6 +5,14 @@ class FakeUrl {
    * */
   private readonly baseUrl: string
 
+  /**
+   * used when the a tab has to navigate to new dApp page
+   *
+   * Swarm Session ID is not requried
+   * it will redirect the current tab
```

Comment:
Some typos here:

used => Used
the a => a
requried => required

Maybe a new line would be nice before the last sentence, also capitalization.